01-exercise-easy/06-counting-digits-in-number/countingDigitsInNumber_Dev.py

```diff
@@ -0,0 +1,11 @@
+def count_digits(n):
+    lista = [0]*10
```

Comment:
É uma boa prática utilizarmos nomes de variáveis mais específicas, neste caso poderia ser por exemplo:
```suggestion
   contagens = [0] * 10
```

Pois sabemos que é uma lista quando aberto um array para armazenar as contagens.